i18n/bundle/bundle.go

```diff
@@ -134,6 +134,25 @@ func (b *Bundle) Translations() map[string]map[string]translation.Translation {
 	return b.translations
 }
 
+// LanguageTags returns the tags of all languages that that have been added.
+func (b *Bundle) LanguageTags() (tags []string) {
+	for k := range b.translations {
+		tags = append(tags, k)
+	}
+	return
+}
+
+// LanguageTranslationIDs returns the ids of all translations that have been added for a given language.
+func (b *Bundle) LanguageTranslationIDs(languageTag string) (iDs []string) {
```

Comment:
Please don't use named return variables for functions that only have one return variable. I would rather see the variable declaration in the function implementation.

Also, "ids" not "iDs"

```go
var ids []string
```